ginlsat/ginlsat.go

```diff
@@ -37,11 +37,12 @@ const (
 )
 
 type LsatInfo struct {
-	Type     string
-	Preimage lntypes.Preimage
-	Mac      *macaroon.MacaroonIdentifier
-	Amount   int64
-	Error    error
+	Type        string
+	Preimage    lntypes.Preimage
+	Mac         *macaroon.MacaroonIdentifier
```

Comment:
```suggestion
```
If we just fill in the hash, this field `Mac` is never used. So I think you can delete it.